AndyTheNerd/Storehaus
# Add timed scheduler for promotion commands

Timer to ask TR staff to use !d bump
Remind people to vote for Storehaus on top.gg

Runs every 12 hrs

To test:
1. Run npm install
2. Switch commented lines in index
3. Either change timer on cron (https://crontab.guru/every-1-minute) or wait for 12 hours
4. See channels and logs show timer was executed
